website/content/docs/install/windows-service.mdx

```diff
@@ -69,6 +69,26 @@ You have two ways to start the service.
 The service automatically starts up during/after boot, so you don't need to
 launch Nomad from the command-line again.
 
+## Increasing the non-interactive desktop heap size
+
+If you run Nomad as a Windows service and you intend to run many allocations
+(e.g., more than 50), you may encounter issues with Nomad plugin processes
+being terminated by Windows with errors such as `exit status 0xc0000142`
```

Comment:
```suggestion
, such as more than 50, you may encounter issues with Windows terminating Nomad plugin processes with errors such as `exit status 0xc0000142`
```

style nits:
- remove "e.g."
- use commas instead of parentheses b/c readers often ignore content inside parentheses
- active voice (remove passive "being terminated by")